### PR TITLE
docs: add FAQ for sleep schedule

### PR DESCRIPTION
**Pull request by Kimchelly**

### CODE REVIEW

File: docs/Hosts/Spawn-Hosts.md [hunk 1]
```diff
@@ -173,6 +166,21 @@ This can also be done through the CLI:
...
 
+#### FAQ
+
+**Q: Can I still manually stop and start my hosts if the schedule is running?**
+A: Yes! Even though Evergreen will automatically stop and start your host on the configured schedule, you can still
+manually stop and start your host whenever you need to. If you do this, it's recommended, you also [set a temporary
+exemption](#temporary-exemptions) to ensure that you can use your host without being accidentally interrupted when the
+sleep schedule tries to automatically stop/start your host.
+
+**Q: Is there a way to get a hostname that doesn't change when the host restarts?**
```
Comment (malikchaya2) [inline docs/Hosts/Spawn-Hosts.md:177]: I think that the "Is there a way to" wording might read as "It is configurable". 
Can we change it to something like: 
`Will the hostname be persistent through restarts?`
or
`Will the hostname change every time the host restarts?`

**Review by malikchaya2 — APPROVED** [review]
LGTM with one request for a wording change

File: docs/Hosts/Spawn-Hosts.md (hunk 1 above)
Comment (Kimchelly) [reply docs/Hosts/Spawn-Hosts.md]: Sure, I updated it to the first one (kind of).

**Review by Kimchelly — COMMENTED** [review]
Could I get a re-review? I added an example so people have a basic template for what it looks like in an SSH config.

File: docs/Hosts/Spawn-Hosts.md [hunk 2]
```diff
@@ -173,6 +166,30 @@ This can also be done through the CLI:
...
+
+**Q: Can I still manually stop and start my hosts if the schedule is running?**
+A: Yes! Even though Evergreen will automatically stop and start your host on the configured schedule, you can still
+manually stop and start your host whenever you need to. If you do this, it's recommended, you also [set a temporary
+exemption](#temporary-exemptions) to ensure that you can use your host without being accidentally interrupted when the
+sleep schedule tries to automatically stop/start your host.
+
+**Q: Will the hostname change between restarts?**
+
+A: You can get a hostname that doesn't change from your unexpirable host. While it's true that EC2 hosts typically do
```
Comment (malikchaya2) [inline docs/Hosts/Spawn-Hosts.md:179]: > You can get a hostname that doesn't change from your unexpirable host.
Do they need to do anything special to request it? If yes, can we add those details? 
If not, can we rephrase it to something like "You _will_ be automatically assigned a...

**Review by malikchaya2 — APPROVED** [review]
LGTM with one request

File: docs/Hosts/Spawn-Hosts.md (hunk 2 above)
Comment (Kimchelly) [reply docs/Hosts/Spawn-Hosts.md:179]: It's automatically assigned, I'll mention it.
